src/locales/translation.js

```diff
@@ -0,0 +1,14 @@
+export default {
+    'Edit': {
+     	en: 'Edit',
+     	es: 'Editar'
+    },
+    'and save to reload.': {
```

Comment:
maybe is better use keys without spaces like `And_Save_To_Reload` or `and_save_to_reload`

Reply:
If you check the link in the source, the examples are like this. We can use it in this way if is better, of course.

Reply:
I understand your point @fcoronelmakingsense but maybe not, I would use the suggested way proposed by library owners.